terraform/gcp/security/gcp-bigquery-dataset-encrypted-with-cmk.tf

```diff
@@ -0,0 +1,36 @@
+# fail
+# ruleid: gcp-bigquery-dataset-encrypted-with-cmk
+resource "google_bigquery_dataset" "fail" {
+  dataset_id                  = "example_dataset"
+  friendly_name               = "test"
+  description                 = "This is a test description"
+  location                    = "EU"
+  default_table_expiration_ms = 3600000
+
+  labels = {
+    env = "default"
+  }
+
+  access {
+    role          = "OWNER"
+    special_group = "allAuthenticatedUsers"
```

Comment:
Is the issue in this case that there is no encryption, or that allAuthenticatedUsers is used here?